corehq/apps/app_manager/templates/app_manager/app_diff.html

```diff
@@ -26,9 +26,11 @@ <h2>{% trans 'MultiMedia Sizes' %}</h2>
       {% include 'app_manager/partials/multimedia_sizes.html' with version=other_app.version %}
     </div>
     <div id="multimedia-sizes-diff">
+      <b>
```

Comment:
Nitpick: it'd be more semantic to use a header here, `h3` or `h4`.

Reply:
https://github.com/dimagi/commcare-hq/pull/25868/commits/4f9470c68276bb8b64d67633dceabe20a76ea7dc